source/reference/mongosync/mongosync-behavior.txt

```diff
@@ -140,16 +140,15 @@ Consistency
 ~~~~~~~~~~~
 
 ``mongosync`` supports :term:`eventual consistency` on the destination
-cluster. Read Consistency is not guaranteed on the destination cluster,
-the source and destination clusters may differ at a given point in
-time. However, if new writes are halted on the source cluster, the
-source and destination clusters will eventually hold the same data.
+cluster. Read consistency is not guaranteed on the destination cluster until 
+commit. Before committing, the source and destination clusters may differ at a 
+given point in time. 
```

Comment:
it would be great to link to the context to-be-added (from this [thread](https://github.com/mongodb/docs-cluster-to-cluster-sync/pull/377/files#r1723941662)) here as well.